src/main/java/core/basesyntax/Colours.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public enum Colours {
```

Comment:
common mistake for enum naming - should be singular
[Style guide](https://mate-academy.github.io/style-guides/java/java.html#s4.8.1-enum-classes)